plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -28,38 +29,69 @@ import (
 
 // the unused capacity is calculated on a scale of 0-10
 // 0 being the lowest priority and 10 being the highest
-func calculateScore(requested, capacity int64, node string) int {
+func calculateScore(requested int64, capacity int64, node string) int {
 	if capacity == 0 {
 		return 0
 	}
 	if requested > capacity {
-		glog.Infof("Combined requested resources from existing pods exceeds capacity on minion: %s", node)
+		glog.Infof("Combined requested resources %d from existing pods exceeds capacity %d on node %s",
```

Comment:
I'm not really sure how helpful this additional data info will be, as one will need to guess what resource it's about. In addition I'm not sure if plain Info won't spam normal logs - we'll probably get over capacity pretty often in case of 0-limit Pods.